supplier_app/tests.py

```diff
@@ -188,18 +172,60 @@ def setUp(self):
         self.client.force_login(self.user_with_eb_social)
         self.companyuserpermission = \
             CompanyUserPermission.objects.create(company=self.company, user=self.user_with_eb_social)
-        self.factory = RequestFactory()
+        self.POST = {
+            'csrfmiddlewaretoken': '67lLxnP0Q0oDIYThiF0z7cEcuLrmJSvT1OJUH0J9RyByLxiMeghEHuGKowoq4bZa',
+            'taxpayer_form-workday_id': '1',
+            'taxpayer_form-business_name': 'EB ARG',
+            'taxpayer_form-cuit': '20-3123214-0',
+            'taxpayer_form-country': 'AR',
+            'taxpayer_form-comments': 'dafsadsffasdf',
+            'taxpayer_form-payment_type': 'dafsadsffasdf',
+            'taxpayer_form-AFIP_registration_file': self.file_mock.name,
+            'taxpayer_form-witholding_taxes_file': self.file_mock.name,
+            'address_form-street': 'San Martin',
+            'address_form-number': '21312',
+            'address_form-zip_code': '123',
+            'address_form-city': 'Mendoza',
+            'address_form-state': 'Mendoza',
+            'address_form-country': 'Argentina',
+            'bankaccount_form-bank_bank_cbu_file': self.file_mock.name,
```

Comment:
`bankaccount_form-bank_bank_cbu_file` should be renamed to `bankaccount_form_bank_cbu_file`

Reply:
That is becaus when I put a prefix on a Django Form then he append that prefix before the field name